pages/dashboard/index.js

```diff
@@ -2,8 +2,15 @@ import React from 'react';
 import { useState } from 'react';
 import PropTypes from 'prop-types';
 import { getSession } from 'next-auth/client';
-import { Grid, Hidden, Button, Drawer, makeStyles, Fab } from '@material-ui/core';
+import { Grid, Hidden, Button, Drawer, Fab, Box } from '@material-ui/core';
+import { useTheme, makeStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+
 import MenuIcon from '@material-ui/icons/Menu';
+import IconButton from '@material-ui/core/IconButton';
```

Comment:
please remove space in line 8. Also try to group mui (material ui) imports together instead of having `classnames` separating them.